lib/widgets/context_menu/twake_context_menu.dart

```diff
@@ -15,19 +17,23 @@ const double _kMinTileHeight = 24;
 /// If you just want to use a normal [TwakeContextMenu], please use [TwakeContextMenuArea].
 
 class TwakeContextMenu extends StatefulWidget {
+  final BuildContext dialogContext;
+  final List<ContextMenuAction> listActions;
```

Comment:
IMO: Should comment a little bit how to use it?

Reply:
Updated